babykicks/src/routes/navigation/navbar.component.js

```diff
@@ -31,17 +32,57 @@ const drawerWidth = 240;
 
 function ResponsiveDrawer(props) {
   const { currentUser } = useContext(UserContext);
+  const { handleSignInClick, handleSignOutClick } = useGoogleAuth();
+
+  const navbarItems = [
+    {
+      to: "",
+      label: "Home",
+      icon: <HomeIcon />,
+      renderDivider: false,
+      handleClick: null,
+    },
+    {
+      to: null,
+      label: currentUser ? "Sign Out" : "Sign In",
+      icon: currentUser ? <LogoutIcon /> : <LoginIcon />,
+      renderDivider: false,
+      handleClick: currentUser ? handleSignOutClick : handleSignInClick,
```

Comment:
this combined with the useGoogleAuth hook means we can remove the multi-purpose component that googleSignIn / googleSignOut components was, and allow:

1. this navbarItems to take in charge of rendering lists and list items
2. the useGoogleAuth hook to take charge in housing the logic behind the google authentication